index.html

```diff
@@ -1,14 +1,168 @@
+                        like a super power. Power to bring ones vision to life. With robustness, speed, scalability and
+                        all the best practices one can have, you develop like a hero. A super hero fullstack developer
+                        in the world of web.</p>
+                </div>
+                <div class="nextdate">
+                    <h3><b>2023.05.01(MON) ~ 03(WED)</b></h3>
+                    <h5>@Web-Hero's next camp, Secure you place now!</h5>
+                </div>
+            </div>
+        </section>
+        <section class="courses" id="cources">
+            <h1>Top Cources</h1>
+            <div class="line"></div>
+            <div class="cards">
+
+                <a href="#">
+                    <div class="cc">
+                        <img src="assets/cards/angular.svg" alt="Angular icon">
+                        <h2>Angular</h2>
+                        <p>Angular is a TypeScript-based, free and open-source web application framework led by the...
+                        </p>
+                    </div>
+                </a>
+                <a href="#">
+                    <div class="cc">
+                        <img src="assets/cards/js.svg" alt="Angular icon">
+                        <h2>Javascript</h2>
+                        <p>JavaScript, or JS, is a programming language that is a core technologies of the World Wide
+                            Web.</p>
+                    </div>
+                </a>
+                <a href="#">
+                    <div class="cc">
+                        <img src="assets/cards/react.svg" alt="Angular icon">
+                        <h2>React</h2>
+                        <p>React is a top front-end JavaScript library for building user interfaces based on components.
+                        </p>
+                    </div>
+                </a>
+                <a href="#">
+                    <div class="cc">
+                        <img src="assets/cards/html5.svg" alt="Angular icon">
+                        <h2>HTML5</h2>
+                        <p>HTML5 is a markup language used for structuring & presenting the World Wide Web.</p>
+                    </div>
+                </a>
+                <a href="#">
+                    <div class="cc">
+                        <img src="assets/cards/css3-alt.svg" alt="CSS3 icon">
+                        <h2>CSS3</h2>
+                        <p>Cascading Style Sheets is a style sheet language used for describing the presentation of the
+                            HTML.</p>
+                    </div>
+                </a>
+
+            </div>
+            <a class="full-list" href="#">See the complete list of cources</a>
+            <a class="join-list" href="#">Join u'r favorite course in 2023</a>
+        </section>
+        <section class="mentors" id="mentors">
+            <h1>Featured Mentors</h1>
```

Comment:
- [ ] You've done a great job of identifying the titles on your page :+1:. However, it's important to note that each page should have at least one, but no more than one, `h1` tag. As such, please review the "index" and "about" pages to ensure that there is only one `h1` tag on each of these pages. By making sure that each page has only one `h1` tag, the website will have a more organized and user-friendly structure, making it easier for visitors to navigate and find the information they need.